frameworks/helloworld/src/main/dist/examples/sidecar.yml

```diff
@@ -36,6 +38,10 @@ pods:
         goal: FINISHED
         cmd: "echo $PLAN_PARAMETER >> hello-container-path/output && grep parameterized < hello-container-path/output"
         resource-set: sidecar-resources
+      sidecar-toxic:
+        goal: FINISHED
+        cmd: "echo 'I'm addicted to you / Don't you know that you're toxic?' && exit 1"
```

Comment:
Could we split the `cmd` over multiple lines to make the `exit 1` at the end easier to notice?